### DISCUSSION TITLE
[WIP] SkeletonIK3D

### DISCUSSION THREAD

**Discussion by GeorgeS2019** [godotengine/godot-proposals#6986] [3D]
<details>
<summary><strong>Two approaches of Godot4 3D IK</strong></summary>

- [ ]  **SkeletonIK3D** (IK **scene tree node**)
> This active development approach
- [ ] Many Bone IK using [**AnimationTree**](https://gist.github.com/fire/1953bd6bb0e9cc3b964796317d23c702) [[[ewbik or Everything-Will-Be-IK](https://github.com/V-Sekai/many_bone_ik)]
> [Needs more testing, use cases - video demo 2023 Feb ](https://youtu.be/6Hu5wAJWBQ0)

</details>

**Comment by AThousandShips** [comment]


**Reply by AThousandShips** [reply]
Just edit your post, you can just update it, no one is asking you to get it right on the first try, but it is a source of annoyance for those who subscribe to the repository and get pinged for a lot of replies on the same post by the same person

**Reply by GeorgeS2019** [reply]
@AThousandShips  simplified.

**Reply by AThousandShips** [reply]
Yes, that doesn't remove the notifications, just *edit* your post, what's preventing you from doing that? Or just track it on some other platform, just brainstorming by adding comments is disruptive 

Please listen, I won't pollute the notifications further now

(My bad my comment accidentally deleted when I was scrolling on my phone)
